api-reference/docstubs/api/microsoft.fast.pi.viewgenerator.v2.profile.models-profile-get.md

```diff
@@ -0,0 +1,78 @@
+---
+title: "Get profile"
+description: "Read properties and relationships of a profile object."
+author: "**TODO: Provide Github Name**"
+localization_priority: Normal
+ms.prod: ""
+doc_type: apiPageType
+---
+
+# Get profile
+
+Namespace: Microsoft.Fast.PI.ViewGenerator.V2.Profile.Models
+
+Read properties and relationships of a [profile](../resources/microsoft.fast.pi.viewgenerator.v2.profile.models-profile.md) object.
+
+## Permissions
+One of the following permissions is required to call this API. To learn more, including how to choose permissions, see [Permissions](/concepts/permissions-reference.md).
+
+|Permission type|Permissions (from most to least privileged)|
+|:---|:---|
+|Delegated (work or school account)|**TODO: Provide applicable permissions.**|
+|Delegated (personal Microsoft account)|**TODO: Provide applicable permissions.**|
+|Application|**TODO: Provide applicable permissions.**|
+
+## HTTP request
+<!-- {
+  "blockType": "ignored"
+}
+-->
+``` http
+GET /user/{userId}/profile
+```
+
+## Optional query parameters
+This method supports some of the OData query parameters to help customize the response. For general information, see [OData query parameters](/graph/query-parameters).
+
+## Request headers
+|Name|Description|
+|:---|:---|
+|Authorization|Bearer {token}. Required|
+
+## Request body
+Do not supply a request body for this method.
+
+## Response
+If successful, this method returns a `200 OK` response code and a [profile](../resources/microsoft.fast.pi.viewgenerator.v2.profile.models-profile.md) object in the response body.
+
+## Examples
+
+### Request
+<!-- {
+  "blockType": "request",
+  "name": "get_profile"
+}
+-->
+``` http
+GET https://graph.microsoft.com/beta/user/{userId}/profile
+```
+
+### Response
+Note: The response object shown here may be truncated for brevity. All of the properties will be returned from an actual call.
```

Comment:
I think this needs to be a proper callout using \>
> **Note:** blah blah blah

Also I think the doc platform has additional callout like this (different from standard MD)
\> [!NOTE]
\> \<note content\>

Reply:
GLOBAL:
Probably commented in your [PR #2](https://github.com/wandeg/microsoft-graph-docs/pull/2) earlier - Let's remove this sentence in the boilerplate for the example response section: "All of the properties will be returned from an actual call."

Reply:
BUG BUG - also check if we had a bug for this already.